wp/plugin/tests/test_wp.ml

```diff
@@ -107,30 +103,27 @@ let suite = [
   "Switch Case Assignments"        >:: test_compare_elf "switch_case_assignments" "SAT!" ~func:"process_status";
   "Switch Cases"                   >:: test_compare_elf "switch_cases" "SAT!" ~func:"process_message" ~check_calls:true;
   "No Stack Protection"            >:: test_compare_elf "no_stack_protection" "SAT!" ~output_vars:"RSI,RAX";
-  "Retrowrite Stub"                >:: test_compare_elf "retrowrite_stub" "UNSAT!"
-    ~inline:true ~to_inline:"__afl_maybe_log";
-  "Retrowrite Stub: imply inline"  >:: test_compare_elf "retrowrite_stub" "UNSAT!" ~to_inline:"__afl_maybe_log";
-  "Retrowrite Stub: inline all"    >:: test_compare_elf "retrowrite_stub" "UNSAT!" ~inline:true;
+  "Retrowrite Stub"                >:: test_compare_elf "retrowrite_stub" "UNSAT!" ~inline:"__afl_maybe_log";
+  "Retrowrite Stub: imply inline"  >:: test_compare_elf "retrowrite_stub" "UNSAT!" ~inline:"__afl_maybe_log";
```

Comment:
I think we can remove the "imply inline" tests, now that we consolidated the inline flag.